### DISCUSSION TITLE
Milestones for documents repo

### DISCUSSION THREAD

**Discussion by scottrigby** [open-gitops/project#17] [Ideas] [answered]
Hi everyone 👋 

We've had really great discussion during the past few [principles meetings](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.kr2ebmvnmiq7). I [mentioned in slack](https://cloud-native.slack.com/archives/C01N77FDHCZ/p1627502780106500), I personally really appreciate the dedication, thoroughness and trust that the group members place in one another to debate important points like these in a mutually respectful way 💯 

My action item was to create milestones in the [documents repo](https://github.com/open-gitops/documents). You can [see all the milestones here](https://github.com/open-gitops/documents/milestones), where I have tied issues, set a theme for each as we discussed, and proposed provisional dates.

As of today's meeting the milestones have changed to suit our new feedback model to accelerate progress.

Let's use this discussion for sorting out these milestones as needed. EmojiOps welcome 😅❤️

**Comment by jlbutler** [comment]
Nice!  Do we want to add milestones for glossary and for language accessibility? I think they could be 0.4 and 0.5 respectively, given we want the content primarily in place before putting a ribbon on those.

**Reply by scottrigby** [reply]
I like that idea. Though in order to "resolve notes section" as it so happens this includes updating the glossary. Maybe we should change the theme of the v0.3.0 milestone to both "resolve notes and glossary"?

OK will move "Ensure accessible language" from v1.0.0 to a new pre-release (v0.4.0), set it for the following week, and update the table above.

**Reply by scottrigby** [reply]
Updated. Looking better?

**Comment by moshloop** [comment]
+0.9,  I would prefer 0.2.0 for just simplifying the titles, and the "catchy" phrasing moving to 0.4.0, but it is also acceptable for 0.2.0, it just makes the bar higher.

**Reply by scottrigby** [reply]
Agreed. Just updated the v0.2.0 milestone description. See the [most recent change](https://docs.github.com/en/communities/moderating-comments-and-conversations/tracking-changes-in-a-comment) above.

**Comment by scottrigby** [comment]
📣 Hi everyone!

📆 The WG co-chair meeting wasn’t able to happen this week, and Principles committee members who are also on the Events committee will be busy reviewing GitOpsCon CFPs. Let's cancel this week's principles meeting, and push our documents repo milestones back by two weeks.

✏️ I'll:
- [x] update the dates above and on the milestones themselves
- [x] Will also update the calendar event 👍 
- [x] note in the [WG Meetings Google doc](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.t0snkkwv4af1)
- [x] and on the [mailing list](https://lists.cncf.io/g/cncf-tag-app-delivery/topic/gitops_wg_moving_this/84646958?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,84646958)

🤝 Plz add your topics for [the next meeting on Aug 11th](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#). Looking forward working with you async until then!

**Comment by scottrigby** [comment, marked as answer]
We have updated these milestones based on feedback and today's meeting. We are moving from the previously proposed multiple MAJOR version zero releases to an RC milestone/feedback model. That is, the GWG/OGO maintainers will create a RC to incorporate all previous feedback and pass to the group (and other interested parties etc) for async feedback as we make changes toward the next RC. This will also be sent to slack and the mailing list.
